index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
That's true, but people don't search for "flutter", they search for "mobile app sdk" or "mobile app framework", etc.

Here's what we look like today: it's descriptive and helpful:

<img width="697" alt="screen shot 2017-03-01 at 10 08 28 am" src="https://cloud.githubusercontent.com/assets/5479/23473986/12fd8dc8-fe67-11e6-8199-665a379492f2.png">